Dockerfile

```diff
@@ -14,11 +14,11 @@ RUN apt-get update && \
 RUN apt-get update && \
     apt-get install software-properties-common=0.92.37.3 -y
 
-# Install git with security updates
+# Install latest git with security updates
 # http://article.gmane.org/gmane.linux.kernel/1853266
 RUN add-apt-repository ppa:git-core/ppa -y && \
     apt-get update -yq && \
-    apt-get install -yq git=1:2.3.5-0ppa1~ubuntu14.04.1
+    apt-get install -yq git
```

Comment:
@chr0n1x latest from that upstream, older, explicit versions are `removed` quite frequently, and its a gigantic pain to maintain